tui/keychainmanagementui/model.go

```diff
@@ -226,18 +234,21 @@ func createKeychain(m *Model) {
 		accessTx.OriginSign(originPrivateKey)
 		ts2 := archethic.NewTransactionSender(client)
 		ts2.AddOnConfirmation(func(nbConf int, maxConf int) {
+			m.showLoading = false
```

Comment:
I think you should use the `AddOnRequiredConfirmation` instead.
But if want to leverage the AddOnConfirmation maybe a progress bar or a counter can be interesting as well.

Reply:
yes definitely, well spotted! 
I fixed that yesterday, but I've probably lost the commit in between the 2 branches 🤷‍♂️